data/repos.yml

```diff
@@ -264,6 +264,12 @@
   team: "#data-products"
   type: Data science
 
+- repo_name: docker-ckan
```

Comment:
This will probably be archived as I have moved the docker code to the ckanext-datagovuk repo